app/code/Magento/Theme/view/frontend/templates/html/pager.phtml

```diff
@@ -87,7 +88,10 @@
 
             <?php if ($block->canShowNextJump()): ?>
                 <li class="item">
-                    <a class="page next jump" title="" href="<?= $block->escapeUrl($block->getNextJumpUrl()) ?>">
+                    <a class="page next jump"
+                       title=""
+                       aria-label="Skip to page <?= $block->escapeHtml($block->getNextJumpPage()) ?>"
```

Comment:
Can you translate this?

Reply:
@blmage can you clarify further what you mean by "translate this"?

Reply:
@bradleybrecher 'Skip to Page' should be translatable with `__()`

Reply:
change pushed & ready for another review